src/controls/combinators/AnyOfControl.test.tsx

```diff
@@ -37,17 +37,16 @@ describe("AnyOf control", () => {
     // Column Name is available in both subschemas
     await screen.findByText("Splitter")
     screen.getByLabelText("Column Name")
-    expect(screen.queryByLabelText("Method Name")).toHaveValue("split_on_year")
+    screen.getByTitle("split_on_year")
+    expect(
+      screen.queryByTitle("split_on_year_and_month"),
+    ).not.toBeInTheDocument()
```

Comment:
The `enum` nested in this `anyOf` turned into a searchable dropdown rather than a text input.

Reply:
The storybook for `anyOf` looks unchanged to me.

Reply:
Yeah I think we've been using enums with length of 1 & default values as union discriminators basically (and not rendering them as fields). I don't think this expectation makes sense prior to this change, and after this change, it kinda does. LGTM